threediresultstyler/test/qgis_interface.py

```diff
@@ -0,0 +1,205 @@
+# coding=utf-8
```

Comment:
Dit hele bestand is geen testbestand. Ziet eruit alsof het weg kan. Copy/paste van een eerdere poging?